### PR TITLE
Plans: update FAQ section

### PR DESCRIPTION
**Pull request by rcanepa**
#### Changes proposed in this Pull Request

* Our current FAQ section below the plans grid is not up to date so the purpose of this PR is to update the copy of every question.

#### Testing instructions

* Run this PR with `yarn start`. 
* Visit `jetpack/connect/store`.
* Go to the bottom of the page and verify that there are 5 questions.
* Verify that links work as expected.
* Verify that the copy is correct (the link to the document is the ticket).

Fixes 1169247016322522-as-1193401773801897

##### Before
image

##### After
image

### CODE REVIEW

File: client/my-sites/plans-features-main/jetpack-faq.jsx [hunk 1]
```diff
@@ -25,45 +25,39 @@ const JetpackFAQ = ( { isChatAvailable, translate } ) => {
...
 				) }
 			/>
 
 			<FAQItem
-				question={ translate( 'Does this work with a multisite network?' ) }
+				question={ translate( 'Why do I need a WordPress.com account?' ) }
 				answer={ translate(
-					"Yes, Jetpack and all of it's paid features are compatible with WordPress Multisite" +
-						' networks, except for Jetpack Backup.'
+					"Many of Jetpack's core features make use of the WordPress.com cloud. In order to make sure everything" +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/jetpack-faq.jsx:42]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Many of Jetpack's core features make use of the WordPress.com cloud. In order to make sure everything works correctly, Jetpack requires you to connect a (free) WordPress.com account. If you don't already have an account you can easily create one during the connection process.' )` *ES Score: 9*

**Review by elliottprogrammer — APPROVED** [review]
Verified the copy is correct. Everything looks good, Code looks good. LGTM!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/4397318

Thank you @rcanepa for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/4397318).

**Pull request merged**